Adds instructions for installing other 9.x versions (#2426)
elastic/docs-content: deploy-manage/deploy/self-managed/install-elasticsearch-docker-basic.md

3. Pull the {{es}} Docker image.

    ::::{tab-set}
    :group: docker
    :::{tab-item} Latest
    :sync: latest
    ```sh subs=true
    docker pull docker.elastic.co/elasticsearch/elasticsearch:{{version.stack}}
    ```
    :::

    :::{tab-item} Specific version
    :sync: specific

>     Replace `<SPECIFIC.VERSION.NUMBER>` with the {{stack}} version number you want. For example, you can replace `<SPECIFIC.VERSION.NUMBER>` with {{version.stack.base}}.

Reviewer: something like

```suggestion
    Replace `<SPECIFIC.VERSION.NUMBER>` with the {{stack}} version number you want. For example, you can replace `<SPECIFIC.VERSION.NUMBER>` with {{version.stack.base}}. You'll use this same version number throughout this tutorial.
```

      - Existence of the claims in the transparency log was verified offline
      - The signatures were verified against the specified public key
    ```
    :::
    
    :::{tab-item} Specific version
    :sync: specific

>     Use the same {{stack}} version number as the Docker image you pulled earlier and replace `<SPECIFIC.VERSION.NUMBER>` with it.

Reviewer: consider: 

```suggestion
    Replace `<SPECIFIC.VERSION.NUMBER>` with the version of the Docker image [that you downloaded](link up to step - needs an anchor).
```

or

```suggestion
    Replace `<SPECIFIC.VERSION.NUMBER>` with the {{stack}} version of [your Docker image](link up to step - needs an anchor).
```
